client/my-sites/plugins/plan-setup/index.jsx

```diff
@@ -42,13 +42,15 @@ module.exports = React.createClass( {
 	},
 
 	componentDidMount() {
+		if ( ! this.props.selectedSite || ! this.props.selectedSite.jetpack || ! this.props.selectedSite.canManage() ) {
+			return;
```

Comment:
We do, see the `render` function — but we still need this check before trying to `runInstall`.